BUILD.md

```diff
@@ -283,7 +283,7 @@ It fixes some source files to add missing `#include`s.
    Single-config generators:
 
    ```
-   cmake --build .
+   cmake --build . -j $(nproc)
```

Comment:
`nproc` is not available on macos.
Since BUILD.md mentions building on Linux/Windows/Mac, it would be helpful to include the  commands for each.

I usually use `sysctl -n hw.logicalcpu` on macos.

Reply:
Yes, I forgot to mention that also. 

`brew install coreutils` will get you `nproc` instead of that malarky.